PayloadPkg/FirmwareUpdate/GetCapsuleImage.c

```diff
@@ -23,8 +23,10 @@
 #include <Library/HobLib.h>
 #include <Library/MediaAccessLib.h>
 #include <Library/BootloaderCommonLib.h>
-#include <Library/FirmwareUpdateLib.h>
 #include <Library/CryptoLib.h>
+#include <Library/FirmwareUpdateLib.h>
+#include <Library/ConfigDataLib.h>
+#include <ConfigDataCommonStruct.h>
 
 #define MM_PCI_BASE(Bus, Device, Function) \
```

Comment:
there is a definition in common header already

Reply:
Updated code to the latest